packages/cached/test/cached-fs-node.ix.ts

```diff
@@ -0,0 +1,78 @@
+import { expect } from 'chai';
+import { createNodeFs } from '@file-services/node';
+import { createCachedFs } from '../src';
+
+describe(`cachedFs with Node's fs`, () => {
+    it('caches statsSync calls', async () => {
+        const filePath = '../../package.json';
```

Comment:
common between all tests. move to describe. Use `require.resolve('../../package.json')`, so it won't depend on the current cwd.

Reply:
fixed